trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.test.ts

```diff
@@ -291,6 +299,54 @@ describe('controllers/utilisation-reports/utilisation-report-reconciliation-for-
       expect(viewModel.premiumPayments.payments[0].isChecked).toEqual(true);
     });
 
+    it("renders the page with 'isFeeRecordCorrectionFeatureFlagEnabled' set to true if the feature flag is enabled", async () => {
+      // Arrange
+      const { req, res } = httpMocks.createMocks({
+        session,
+        params: {
+          reportId,
+        },
+        query: {
+          matchSuccess: 'true',
```

Comment:
Is this needed? Can we omit this query param?

Same as the FF not enabled case below